lib/utils/index.js

```diff
@@ -384,5 +426,48 @@ module.exports = {
         cb(node.arguments[0])
       }
     }
+  },
+
+  iterateProperties (node, scopeNames, cb) {
```

Comment:
Also, what do you think about calling `scopeNames` -> `groups` instead? Scope is actually the same so being semantically correct we'd call it rather a group. At least I think so